modules/installation-osp-setting-cloud-provider-options.adoc

```diff
@@ -0,0 +1,65 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-installer-user.adoc
+
+[id="installation-osp-setting-cloud-provider-options_{context}"]
+= Setting cloud provider options
+
+Optionally, you can edit the cloud provider configuration for your cluster. The cloud provider configuration controls how {product-title} interacts with {rh-openstack-first}.
+
+For a complete list of cloud provider configuration parameters, see link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
+
+.Prerequisites
+
+* Have manifest files generated for your cluster.
+** If you don't have manifest files, generate them by running `openshift-install --dir <destination_directory> create manifests` from a command line.
+
+.Procedure
+
+. In a text editor, open the cloud-provider configuration manifest file. For example:
++
+[source,terminal]
+----
+$ vi openshift/manifests/cloud-provider-config.yaml
+----
+
+. Modify the options based on link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
++
+Configuring Octavia for load balancing is a common case. For example:
++
+[source,yaml]
+----
+#...
+[LoadBalancer]
+use-octavia=true <1>
+lb-provider = "amphora" <2>
+floating-network-id="d3deb660-4190-40a3-91f1-37326fe6ec4a"<3>
+floating-subnet-id="4a2fe995-84d4-4eac-8bcf-c5624f63dba0"
+subnet-id="e3d26bd3-0b6b-4c4d-b184-71c5be59b8b2"
+#...
+----
+<1> This property enables Octavia integration.
+<2> This property sets the Octavia provider that your load balancer uses. It accepts `"ovn"` or `"amphora`" as values. If you choose to use OVN, you must also set `lb-method` to `SOURCE_IP_PORT`.
+<3> This property is required if you want to use multiple external networks with your cluster. The cloud provider creates floating IP addresses on the network that is specified here.
+
+[IMPORTANT]
+====
+Prior to saving your changes, verify that the file is structured correctly. Clusters may fail if properties are not placed in the appropriate section.
+====
+
+. Save the changes to the file and proceed with installation.
```

Comment:
In the preview this sentence is preceded by the index n.1 when it should n.3 I think